cli/flox/src/commands/build.rs

```diff
@@ -32,13 +33,16 @@ pub struct Build {
     #[bpaf(external(dir_environment_select), fallback(Default::default()))]
     environment: DirEnvironmentSelect,
 
-    #[bpaf(long, hide)]
-    nixpkgs_url: Option<Url>,
-
     #[bpaf(external(subcommand_or_build_targets))]
     subcommand_or_targets: SubcommandOrBuildTargets,
 }
 
+#[derive(Debug, Clone, Bpaf)]
+enum BaseCatalogUrlSelect {
+    NixpkgsUrl(#[bpaf(long("nixpkgs-url"), argument("url"), hide)] Url),
+    Stability(#[bpaf(long("stability"), argument("stability"))] String),
```

Comment:
What effect, if any, does this have on manifest builds?

How will it be documented in a way that users will understand?

Reply:
stabilities have no effect on manifest builds.
It is however unfortunate that manifest builds and expression builds share the same outside interface.